src/p2/StudentTest.java

```diff
@@ -0,0 +1,22 @@
+package p2;
+
+public class StudentTest {
+
+    public static void main(String[] args) {
+        Student student1 = new Student(123, "Anna", "Ivanova", "Economics",
+                5, 4, 4);
+        Student student2 = new Student(124, "Anastasiya", "Petrova", "Economics",
+                5, 5, 4);
+        Student student3 = new Student(125, "Mariya", "Tkachenko", "Economics",
+                3, 3, 4);
+
+        double average1 = ((double) (student1.averageScoreInMathematics + student1.averageScoreInEconomics + student1.averageScoreInForeignLanguage) / 3);
```

Comment:
It's needed to create a method that calculates average score because we do the same actions many times. It violates DRY principle